auth/scoperesolver.js

```diff
@@ -219,10 +230,11 @@ class ScopeResolver extends events.EventEmitter {
 
   /** Update lastDateUsed for a clientId */
   async _updateLastUsed(clientId) {
-    let client = await this._Client({clientId});
+    let client = await this._Client.load({clientId});
```

Comment:
yeah... really stupid bug... I suspect as a result the auth server having been crashing every 6 hours or so... interesting :)